_config.yml

```diff
@@ -1,5 +1,6 @@
 ---
-title: ChooseALicense.com
+title: Choose a License
+description: A site to provide non-judgmental guidance on choosing a license for your open source project
```

Comment:
Maybe shorten this to ~ "Non-judgmental guidance on choosing a license for your open source project"?